docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3594,34 +3594,60 @@ opportunity for duplication. Since this mechanism occurs within the
 simulation, there is a natural maximum that we will impose in the
 noise function.
 
-Guardian based duplication is further divided here into two types: simulants
-younger than 18 and not in college GQ (<18), and those at college GQ less than 24 (<24).
-
-For simulants younger than 18 and not in college GQ, the maximum duplication rate will
-be calculated based on those who have a guardian living at a different address in the sim.
-
-The user can then pick a rate of duplication between 0 and 100%. If the value selected
-is higher than the calculated maximum rate in the sim, a warning will be issued to users
-explaining that the selected rate is greater than the maximum available.
+Guardian-based duplication is applied to three mutually exclusive categories of
+simulants based on age and GQ status: Simulants younger than 18  (<18) and not
+in GQ; simulants 18-23 (18 <= age < 24) and not in GQ; and simulants under 24
+(<24) and in GQ.
+
+For each of the three categories of simulants, the maximum duplication rate will
+be calculated based on those who have a guardian living at a different address
+in the sim. Note that all simulants in *college* GQ are initialized with a
+guardian living at a different address, but this is not true for simulants in
+other types of GQ, so all three maximum duplication rates will be less than
+100%.
+
+The user can then pick a rate of duplication between 0 and 100% **for each of
+the three categories of simulants**. A default duplication rate of 5% will be
+selected for each of the three categories of simulants. That is, each simulant
+under 24 is duplicated at a guardian's address with default probability 0.05,
+and there should be three user parameters for overriding this probability, one
+for each simulant category. The duplicated row should have the same
+simulant-specific attributes as the original, such as name and birth date, but
+different address-specific attributes such as address fields and relation to
+reference person.
```

Comment:
Sure, I can change "address-specific" to "household-specific."

As far as know, we have **not** specified what the relation to reference person should be when duplicated at a guardian's address. Is there an easy way to figure this out based on how guardians were assigned? (Side note: What is the relation to reference person for someone living in group quarters?)

I hadn't thought any of this through very carefully -- I just realized that we hadn't specified what the duplicate row should look like, so I quickly added something. Seems like it probably needs more detail...

Reply:
I just added a to-do to specify the relation to reference reference person in case we don't figure this out before I leave on Wednesday.

Reply:
> What is the conceptual difference between "household" and "address" in our simulation? Are there any cases where these are not the same thing?

At a single point in time, they are always the same thing, if you consider "address" to mean "address_id." If you consider "address" to mean the actual address string, it is technical possible though extremely unlikely that multiple households could have the same address due to a random collision.

Over time, they are different because a household can move as a unit and change address.

> What is the relation to reference person for someone living in group quarters?

We have special values in this column for "Institutionalized GQ pop" and "Noninstitutionalized GQ pop". We should probably change the names of these, and [document them](https://pseudopeople.readthedocs.io/en/latest/datasets/index.html#id1)!

Reply:
As for the original question: if it's not already specified somewhere, I think we can do something simple like use "Other relative" for all guardian-duplicated rows.

Reply:
I went ahead and specified that relation to reference person should be "Other relative" in the row that's duplicated in the guardian's household.